src/server/graphql/rootQuery.ts

```diff
@@ -4,17 +4,19 @@ import {getUserId} from 'server/utils/authorization'
 import verifiedInvitation from 'server/graphql/queries/verifiedInvitation'
 import rateLimit from 'server/graphql/rateLimit'
 import getRethink from 'server/database/rethinkDriver'
+import massInvitation from 'server/graphql/queries/massInvitation'
 
 export default new GraphQLObjectType({
   name: 'Query',
   fields: () => ({
     viewer: {
       type: User,
-      resolve: (source, args, {authToken, dataLoader}) => {
+      resolve: (_source, _args, {authToken, dataLoader}) => {
```

Comment:
Any reason you decided to rename these?

Reply:
ts protocol is to underscore unused args